lighthouse/environment/src/lib.rs

```diff
@@ -182,7 +182,18 @@ impl<E: EthSpec> EnvironmentBuilder<E> {
 
             // Create the necessary directories for the correct service and network.
             if !dir.exists() {
-                create_dir_all(dir).map_err(|e| format!("Unable to create directory: {:?}", e))?;
+                let res = create_dir_all(dir);
+
+                // If the directories cannot be created, warn and disable the logger.
+                if res.is_err() {
```

Comment:
I could only trigger an error here if the `validators` directory already existed with the wrong permissions (or the LH user had the wrong permissions). In all cases I'd try, LH would exit anyway since it could never create/open the necessary files. Thus I'm not really sure if this check here is necessary, but I left it in just in case there's a future edge case which may trigger it.